ql/src/java/org/apache/hadoop/hive/ql/exec/vector/VectorMapOperator.java

```diff
@@ -807,6 +807,12 @@ private boolean deliverVectorizedRowBatch(Writable value) throws HiveException {
       }
     }
     oneRootOperator.process(value, 0);
+
+    if (value != null) {
```

Comment:
does it mean that comment above the method declaration is not valid?
````
   * The Vectorized Input File Format reader has already set the partition column
   * values, reset and filled in the batch, etc.
````

Reply:
I see a reset happening in VectorizedParquetRecordReader -
https://github.com/apache/hive/blob/f78ca5df80c0bcb566f0915cda65112268df492c/ql/src/java/org/apache/hadoop/hive/ql/io/parquet/vector/VectorizedParquetRecordReader.java#L400

It doesn't seem to happen for ORC files - https://github.com/apache/hive/blob/f78ca5df80c0bcb566f0915cda65112268df492c/ql/src/java/org/apache/hadoop/hive/ql/io/orc/VectorizedOrcInputFormat.java#L122